static/js/settings_org.js

```diff
@@ -976,10 +976,10 @@ exports.build_page = function () {
         }
         if (icon) {
             url = '/json/realm/icon';
-            spinner = $('#icon-spinner-background');
-            upload_text =  $('#realm_icon_upload');
-            delete_button = $('#realm_icon_delete_button');
-            error_field = $("#realm_icon_file_input_error");
+            spinner = $('#realm-icon-upload-widget .upload-spinner-background');
+            upload_text =  $('#realm-icon-upload-widget .settings-page-upload-text');
+            delete_button = $('#realm-icon-upload-widget .settings-page-delete-button');
```

Comment:
`.settings-page-upload-text` and friends feel to verbose for this. I know this class existed before your changes but I think its a good place to refactor sometime after the 2.2 release.

Reply:
@aero31aero yeah I'm also thinking about the name change for `settings-page-upload-text` to `image-upload-text`.